src/Registration/Common/Perform2DTranslationRegistrationWithMeanSquares/Code.py

```diff
@@ -27,95 +27,70 @@
           "<outputImagefile> <differenceImageAfter> <differenceImageBefore>")
     sys.exit(1)
 
-fixedImage = sys.argv[1]
-movingImage = sys.argv[2]
-outputImage = sys.argv[3]
-differenceImageAfter = sys.argv[4]
-differenceImageBefore = sys.argv[5]
+fixedImageFile = sys.argv[1]
+movingImageFile = sys.argv[2]
+outputImageFile = sys.argv[3]
+differenceImageAfterFile = sys.argv[4]
+differenceImageBeforeFile = sys.argv[5]
+
 
 Dimension = 2
-PixelType = itk.F
+PixelType = itk.ctype('float')
 
 FixedImageType = itk.Image[PixelType, Dimension]
```

Comment:
`FixedImageType` and `MovingImageType` should be deduced from the loaded image if that is necessary, or these types could be removed. Same thing for `Dimension`: Dimension = fixedImage.GetImageDimension()

Reply:
In this case, since the filters involved are only available for floating point images, we have to specify the desired pixel type to have a floating point pixel.  Good point on the `fixedImage.GetImageDimension()` -- added.